app/main.py

```diff
@@ -1,6 +1,29 @@
+import os
+import requests
+
+API_KEY = os.environ.get("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(
+        url=URL,
+        params={
+            "key": API_KEY,
+            "q": CITY
+        }
+    ).json()
```

Comment:
what happens if you get an error in response?


I think you should add exception handling